src/main/java/httpserver/http/route/requestmethod/OptionsMethodHandler.java

```diff
@@ -0,0 +1,20 @@
+package httpserver.http.route.requestmethod;
+
+import httpserver.http.StatusCode;
+import httpserver.http.response.ResponseBuilder;
+
+public class OptionsMethodHandler extends ResponseBuilder {
+
+    public String getResponse(String requestPath) {
```

Comment:
Would expect the signature of this method to be:

```
public Response getResponse(Request request)
```

Two reasons: (1) `Request` and `Response` are at the same level of abstraction, and (2) it will give you more flexibility to update this method without having to change the type signature.

Reply:
Thanks Matt. I'll change this